exporter.go

```diff
@@ -9,7 +9,7 @@ import (
 
 	"github.com/kr/beanstalk"
 	"github.com/prometheus/client_golang/prometheus"
-	"github.com/prometheus/common/log"
+	log "github.com/sirupsen/logrus"
```

Comment:
what is the reason to change this?

Reply:
Wanted to override the log level globally, but found I can do this with `log.Base().SetLevel(...)` now. Will revert.